bucket/ripme.json

```
@@ -1,13 +1,13 @@
 {
     "version": "1.7.95",
     "description": "Downloads image albums from various websites in bulk.",
-    "homepage": "https://github.com/RipMeApp/ripme",
+    "homepage": "https://github.com/ripmeapp2/ripme",
     "license": "MIT",
     "suggest": {
         "JRE": "java/temurin8-jre"
     },
-    "url": "https://github.com/RipMeApp/ripme/releases/download/1.7.95/ripme.jar",
-    "hash": "008201e406f401b27248277a4188f26203bb9da0170872de900125f8a6c8b558",
+    "url": "https://github.com/ripmeapp2/ripme/releases/download/2.1.7/ripme-2.1.7-29-b080faae.jar",
+    "hash": "afb0b57cf726abab89943f69345fdffd7dae0ae813ca9ec07f40ac1f65d2413e",
```

```suggestion
    "url": "https://github.com/ripmeapp2/ripme/releases/download/2.1.7/ripme-2.1.7-29-b080faae.jar#/ripme.jar",
    "hash": "afb0b57cf726abab89943f69345fdffd7dae0ae813ca9ec07f40ac1f65d2413e",
    "pre_install": "Set-Content \"$dir\\ripme.cmd\" '@start javaw.exe -jar \"%~dp0ripme.jar\" %*' -Encoding Ascii",
```

- Using `#/` to change the filename of the downloaded URL.